package.json

```diff
@@ -34,12 +34,13 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "event-stream": "3.3.4",
     "gulp-concat": "2.6.1",
     "gulp-footer": "2.0.1",
     "gulp-header": "2.0.5",
     "jsesc": "2.5.1",
     "lodash.template": "^4.4.0",
+    "map-stream": "0.0.7",
+    "stream-combiner": "^0.2.2",
```

Comment:
Could you please explain why you allow for a minor wildcard on _stream-combiner_ but specifically pin _map-stream_? Pinning a version, given the _event-stream_ fiasco, may be preferable, although that does bring with it dependency bloat if other minor/patch versions of these dependencies are used throughout one's project.

Reply:
Initially I just did an "npm install" of both and this is how they were generated. I then checked against event-stream's package.json and saw that they were using them the same way and left them as they are (as I aimed to recreate that).

So the truth is I haven't weighed the pros and cons of pinning them, and I feel I lack the knowledge to do this. I am open to changing either of them.

Reply:
_map-stream_ and _stream-combiner_ were last updated two and three years ago, respectively. Given the infrequency of new versions, my preference would be to pin the versions, and we'll explicitly deal with updates in the future. I don't foresee the dependency bloat here as we would see with more frequent package releases. 

This is particularly interesting as it relates to the _event-stream_ fiasco (note that _angular-template-cache_ skirted this issue as it pinned 3.3.4 and never updated to 3.3.6).